packages/hint/docs/about/GOVERNANCE.md

```diff
@@ -146,27 +147,29 @@ items and commit to being a part of the community for the long-term.
 An individual is invited to become a TSC member by existing TSC members.
 A nomination will result in discussion and then a decision by the TSC.
 
-#### TSC Meetings
+#### Meetings
 
-There are no recurrent TSC meetings, they are scheduled when required,
-at a time that works for the TSC members, and using tools that enable
-participation by the community. The meeting is run by a designated
-moderator approved by the TSC.
+The webhint team tries to meet every 2 weeks (length of a sprint)
```

Comment:
This bit about sprint meetings seems outside the scope of governance/TSC. Curious why this was changed, as I would vote for removing or reverted to previous text.

Reply:
We do try to meet every couple weeks and I believe it's important for the community to know that happens in case they want to join and/or observe. Other projects that mention meetings:

* [eslint](https://github.com/eslint/eslint/blob/d6c112289f0f16ade070865c8786831b7940ca79/docs/maintainer-guide/governance.md#tsc-meetings)
* [babel](https://github.com/babel/notes): There isn't a governance document
* [node.js](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#tsc-meetings)
* [OpenJS CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#team-meetings)

Reply:
Yes, but it looks like all of these are for TSC meetings about governance, not sprint meetings. For example:

> Items are added to the Team agenda that are considered contentious or are modifications of governance, contribution policy, Team membership, or release process.
> The intention of the agenda is not to approve or review all patches; that should happen continuously on GitHub and be handled by the larger group of Collaborators.

Reply:
> **Babel:** To setup, we make a PR on babel/notes and then setup a doodle for availability each week (and we make a Slack call or Google Hangouts for the chat). Currently, we usually do video chat on Mondays at 10:00 PM EST.
> 
> If we don't meet, I usually post a summary of what's been going on (notable issues/PRs, etc).

The OpenJS one doesn't have a TSC and it's open to anyone to participate (and the document explains how/where it is run). We don't review PRs in those meetings. We've been doing planning and coordination with people not in the TSC. It is important for anyone interested to know how/when the team decides to work on something next (and even participate).

Reply:
OK.